etc/ci.sh

```diff
@@ -82,6 +82,41 @@ do
         exit 1
     else
         echo "All packages were built succesfully"
+
+        # also skip itc because it requires xgap package
+        rm -rf itc*
+        # also skip xgap because it should be loaded in other way
+        rm -rf xgap*
+
+        cd ..
+        # Load GAP (without packages) and save workspace to speed up test
+        # save names of all packages into a file to be able to iterate over them
```

Comment:
I read these two lines as one sentence and was confused. Perhaps end the first line with a period `.`, and start the second line as `Also, save names...` ?